tests/test_zapier.py

```
@@ -0,0 +1,91 @@
+    webhook_urls = [
+        "https://hooks.zapier.com/hooks/standard/21128362/d382d71573254c1693fdf743d80b5bd9/", 
+        "https://hooks.zapier.com/hooks/standard/21128362/f0fa4b54f70a48efaf19ff32e7d09b3d/"
+    ]
+
+    deliverer = ZapierDeliverer(webhook_urls=webhook_urls, per_lead_insights=insights)
+
+    response = deliverer.deliver(md5_list)
+
+    # will return False if one of the hooks above unsubscribes aka turned off/deleted. Even if it resubscribes,
+    # the webhook url will be different so it will still return False, Can remove this test if it is not needed.
+    assert response == True
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if response:` for truth checks

Replace with `response`

(E712)